.eslintrc

```diff
@@ -15,6 +15,13 @@
     "node": true
   },
   "rules": {
-    "prettier/prettier": "error"
+    "import/no-extraneous-dependencies": "off",
+    "jsx-a11y/anchor-is-valid": "off",
+    "no-undef": "off", // throw error on graphql queries
```

Comment:
Can you explain this reason some more? `no-undef` is a really really *really* useful lint rule to have :)

Reply:
I agree that it's super useful to have, but right now in the project we don't import graphql when we make a query. I'm assuming that Gatbsy imports the module when it builds the project. This causes the linter to throw an error whenever we make a gql query, so I left it out for now until I can come up with a solution

Reply:
Ah I see. We can solve that problem by adding a global variable definition to the eslint config. See https://eslint.org/docs/user-guide/configuring#specifying-globals for full details, but just adding this snippet to our .eslint file should do the trick:

```diff
 {
+  "globals": {
+    "graphql": false
+  }
 }
```